vendor/engines/projects/app/controllers/projects/projects_controller.rb

```diff
@@ -0,0 +1,37 @@
+module Projects
+
+  class ProjectsController < ApplicationController
+
+    admin_tab :all
+    before_filter { @active_tab = "admin_projects" }
+
+    load_and_authorize_resource
+
+    def index
+      @projects = current_facility.projects.paginate(page: params[:page])
```

Comment:
If you used `load_and_authorize_resource through: :current_facility`, then you could say `@projects = @projects.paginate(...)`. Cancan has some nice loading behavior when you configure it well (most of nucore is not set up to use it well).